integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -0,0 +1,37 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+const stubRequestLink = (): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPattern: '/send-legal-mail/link/email',
+      bodyPatterns: [{ matchesJsonPath: '$[?(@.email =~ /.*/i && @.sessionID =~ /.*/i)]' }],
```

Comment:
We're not actually going to send the session ID as the link is not tied to the session - we tried that in the PoC but then decided not to, but I didn't mention it on this ticket (my bad!)

Reply:
I've removed sessionId 👍